src/scripts/main.js

```diff
@@ -1,3 +1,131 @@
 'use strict';
 
 // write code here
+
+const table = document.getElementsByTagName('table');
+
+const people = table[0].children[1].children;
+
+table[0].children[0].children[0].children[0].addEventListener('click',
```

Comment:
try to avoid so many nested children call (it's not clear what is it)
anyway better to use [eventDelagation](https://javascript.info/event-delegation) in this case